docs/support.rst

```diff
@@ -0,0 +1,352 @@
+
+J Cube
+======
+
+J Cube are a developer of 3D software and cloud services.
+
+Multiverse for Maya
+-------------------
+
+`Multiverse <https://j-cube.jp/solutions/multiverse/>`_ is a USD plugin for Maya with a range of pipeline features.
+
+Muse
+----
+
+`Muse <https://j-cube.jp/solutions/muse/>`_ is a standalone USD Editor
+
+Maxon
+=====
+
+Maxon are a developer of 3D content creation, rendering and motion graphics software.
+
+Cinema 4D
+---------
+
+`Cinema 4D <https://www.maxon.net/en/cinema-4d>`_ is a general purpose 3D package.
+
+Redshift
+--------
+
+`Redshift <https://www.maxon.net/en/redshift>`_ is a GPU-accelerated path-traced renderer.
+
+Redshift supports USD and is also available as a Hydra delegate for rendering integration.
+
+ZBrush
+------
+
+`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.
+
+`ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_
+
+
+NVIDIA Omniverse
+=================
+
+`NVIDIA Omniverse <https://www.nvidia.com/en-us/omniverse/>`_ is a platform for creating and operating metaverse applications.
+It is based on USD.
+
+`NVIDIA USD Documentation <https://www.nvidia.com/en-us/omniverse/usd/>`_
+
+Omniverse also adds USD connectors to many application, `listed here <https://docs.omniverse.nvidia.com/con_connect/con_connect/connecting-to-omniverse.html>`_.
+Some of the applications are:
+
+* Archicad
+* Character Creator
+* Creo
+* iClone
+* ParaView
+* Revit
+* Rhino
+* SketchUp
+
+Please see the `list <https://docs.omniverse.nvidia.com/con_connect/con_connect/connecting-to-omniverse.html>`_  for a
+fuller range of connectors, as there are more than listed here.
+
+Shapr3D
+=======
+
+`Shapr3D <https://www.shapr3d.com>`_ is a CAD application that supports USD export on iPads, Mac and Windows.
+
+
+SideFX Houdini
+==============
+
+`Houdini <https://www.sidefx.com/products/houdini/>`_ is a 3D package with a focus on procedural content and effects.
+
+Houdini includes `Solar <https://www.sidefx.com/products/houdini/solaris/>`_ to create composed USD content.
+
+`SideFX USD Documentation <https://www.sidefx.com/docs/houdini/solaris/usd.html>`_
+
+SynthEyes
+=========
+
+`SynthEyes <https://www.ssontech.com>`_ is a match move application.
+
+Tilt Brush
+==========
+
+`Tilt Brush <https://www.tiltbrush.com>`_ is a VR painting application.
+
+Unity
+=====
+
+`Unity <https://unity.com>`_ is a real time 3D engine and editor.
+It includes an `open source USD Package <https://github.com/Unity-Technologies/usd-unity-sdk>`_.
+
+`Unity USD Documentation <https://docs.unity3d.com/Packages/com.unity.formats.usd@3.0/manual/index.html>`_
+
+Unreal Engine
+=============
+
+`Unreal Engine <https://www.unrealengine.com/en-US>`_ is a real time 3D engine and editor.
+
+`Unreal USD Documentation <https://docs.unrealengine.com/5.0/en-US/universal-scene-description-usd-in-unreal-engine/>`_
+
```

Comment:
I think usdtweak is worth adding too, since it's becoming a very feature rich USD focused editor.

```suggestion

Usdtweak
=============

`usdtweak <https://github.com/cpichard/usdtweak>`_ usdtweak is a free and open source editor for USD.


```